### PR TITLE
Fix/tweaks

### PR DESCRIPTION
**Pull request by adekbadek**
Some tweaks I've added in order for SSL to work with additional sites, and for the scripts to work on my macOS.

### CODE REVIEW

File: bin/init-additional-site.sh [hunk 1]
```diff
@@ -30,15 +30,13 @@ if [ ! -f "/var/www/additional-sites-html/$SITE_NAME/wp-config.php" ]; then
...
 
 	echo "Setting other wp-config.php constants..."
-	wp config set WP_DEBUG true --raw --type=constant
-	wp config set WP_DEBUG_LOG true --raw --type=constant
-	wp config set WP_DEBUG_DISPLAY false --raw --type=constant
-	wp config set SCRIPT_DEBUG true --raw --type=constant
-	wp config set WP_AUTO_UPDATE_CORE true --raw --type=constant
-	wp config set AUTOMATIC_UPDATER_DISABLED true --raw --type=constant
-	wp config set WP_ENVIRONMENT_TYPE local --type=constant
-	wp config set WP_CACHE true --type=constant
```
Comment (leogermani) [inline bin/init-additional-site.sh:40]: Why remove WP_CACHE?

Comment (leogermani) [reply bin/init-additional-site.sh:40]: This is batcache

Comment (naxoc) [reply bin/init-additional-site.sh:40]: I added it back 👍

**Comment by naxoc** [comment]
Thank for this @adekbadek ! Works like a charm, so I'll merge it :)

**Pull request merged**